pkg/cli/run.go

```diff
@@ -218,6 +218,10 @@ func (s *Run) Run(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	existingApp, _ := c.AppGet(cmd.Context(), s.Name)
+	if existingApp != nil && !s.Update {
+		return fmt.Errorf("appinstances.internal.acorn.io \"%s\" already exists", s.Name)
```

Comment:
```suggestion
		return fmt.Errorf("app \"%s\" already exists", s.Name)
```

appinstance is an internal CRD, we dont expose it directly to users. `apps.api.acorn.io` is the actual user facing API. This is why we have two apiGroups: `api.acorn.io` and `internal.acorn.io` in general.

I suggest dropping the api group completely because this is returned directly to the user from the cli and I don't think having the fully qualified name adds any value.